src/fabric/rebar.config

```diff
@@ -1,15 +1,2 @@
-% Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
I'm conflicted on removing licence header here, especially mixed with unrelated changes.

Reply:
The other config files which configure coverage don't have license headers

Reply:
Yes I understand it's inconsistent, but we want it to be at least a separate change so it'll be revertible without messing with unrelated changes, in case, say, it's legally should be other way around and it is all the other configs should have the licence header.